lua/conform/formatters/rustywind.lua

```diff
@@ -0,0 +1,9 @@
+---@type conform.FileFormatterConfig
+return {
+  meta = {
+    url = "https://github.com/avencera/rustywind",
+    description = "Inspired by Ryan Heybourn's headwind vscode plugin. This is a CLI tool that will look through your project and sort all Tailwind CSS classes. It will also delete any duplicate classes it finds.",
```

Comment:
Could you change this description to be `A tool for formatting Tailwind CSS classes.`? This is the string the will appear in the list of formatters.

Reply:
Sure thing. Updated in 2da69125ec016799d91e7bb3fcacd029a379b4cd.